ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/js/editDialog.js

```diff
@@ -79,6 +94,108 @@
         }
     }
 
-    Utils.initializeEditDialog(EDIT_DIALOG)(handleAsyncSubmissionAndThankYouOption, handleSubmitAction);
+    function loadSubmitActionSubDialogClientLibs(dialog) {
+        var containerSubmitAction = dialog.find(CONTAINER_SUBMITACTION)[0];
+        containerSubmitAction.on("change", function(){
+            loadRestEndPointDialogClientlibs(dialog);
```

Comment:
It doesnt look like you are loading client libraries, please re-name the function appropriately

Reply:
changed to registerClientlibs